compiler/src/scala/quoted/runtime/impl/QuotesImpl.scala

```diff
@@ -1695,6 +1695,8 @@ class QuotesImpl private (using val ctx: Context) extends Quotes, QuoteUnpickler
           dotc.core.Types.decorateTypeApplications(self).appliedTo(targ)
         def appliedTo(targs: List[TypeRepr]): TypeRepr =
           dotc.core.Types.decorateTypeApplications(self).appliedTo(targs)
+        def asSeenFrom(pre: TypeRepr, clazz: Symbol): TypeRepr =
+          self.asSeenFrom(pre, clazz)
```

Comment:
We already expose `memberInfo` which calls asSeenFrom, in what situation would you need the raw asSeenFrom?

Reply:
@liufengyun ?

Reply:
If we expose `symbol.info`, without `asSeenFrom`, the info cannot be safely used.

Reply:
symbol.info is the info of the symbol itself, I don't see what's unsafe about it, and if you want to see the info from some specific prefix you can do `prefix.memberType(symbol)`. Are there situations where you need to use `asSeenFrom` directly instead?

Reply:
> symbol.info is the info of the symbol itself, I don't see what's unsafe about it, if you want to see the info from some specific prefix you can do prefix.memberType(symbol). Are there situations where you need to use asSeenFrom directly instead?

The concern is that users use `symbol.info` without rebasing, resulting in error-prone macros. I see that we already have `TypeEpr.memberType`, thus we don't need `asSeenFrom`. However, it is good to add the following doc to `Symbol.info`:

> Warning: `Symbol.info` may contain parameterized types, such as class type parameter, types that depend on `this`, etc. The method call `prefix.memberType(symbol)` can be used to get the concrete type of the symbol relative to a prefix.